### DISCUSSION TITLE
Default options are too opinionated

### DISCUSSION THREAD

**Discussion by eloytoro** [starship/starship#5220] [Q&A]
Just wanted to say that I love this tool, it's simple and fast, configuring it is easy

However, the defaults seem to be very biased towards a predefined config, i had to tweak too many defaults just to remove `in ` and `on ` prefixes that seemed out of place, please consider making this more generic

**Comment by mickimnet** [comment]
Did you check out the [Presets](https://starship.rs/presets/)? Also, available via `starship preset --list`. You can use the presets with `starship preset bracketed-segments -o ~/.config/starship.toml`.

**Comment by eloytoro** [comment]
Well presets are also very opinionated

**Comment by mickimnet** [comment]
Maybe, and you can select a preset without the prefixes.
